aspnetcore/blazor/host-and-deploy/configure-linker.md

```diff
@@ -2,12 +2,12 @@
 title: Configure the Linker for ASP.NET Core Blazor
 author: guardrex
 description: Learn how to control the Intermediate Language (IL) Linker when building a Blazor app.
-monikerRange: '>= aspnetcore-3.1'
+monikerRange: '>= aspnetcore-3.1 && <=aspnetcore-5.0'
 ms.author: riande
 ms.custom: mvc
 ms.date: 05/19/2020
 no-loc: ["ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
-uid: blazor/host-and-deploy/configure-linker
+uid: blazor/host-and-deploy/configure-trimmer
```

Comment:
The runtime team made a conscious decision to call this trimmer. How do we feel about breaking bookmarks?

Reply:
I'll need to do perhaps a touch of work on that. Rick prefers stability in physical locations (URLs) and UIDs. I prefer going ahead and changing them. There are pros and cons of both approaches. Not changing helps to preserve history and requires fewer changes ... but not changing makes a mess of a repo with topics that have virtually irrelevant UIDs and physical locations (and URLs) that don't match where they lay in the ToC. So ...... it's not a bed of roses either way. :smile:

My hours are burned for the week ... is it ok to pick this up on Monday morning?